debian/builds

```diff
@@ -0,0 +1,8 @@
+#! /bin/bash
+
+# cni
+go run build.go -arch amd64 -distro_name xenial -package kubernetes-cni -version 0.0.1 -revision 00
+
+# 1.4.0-beta.0
+go run build.go -arch amd64 -distro_name xenial -package kubelet -version 1.4.0-beta.0 -revision 00
```

Comment:
You can do this for `arm` and `arm64` as well when the `DependsOverride` is in place.